addon/templates/components/bootstrap-checkbox.hbs

```diff
@@ -3,9 +3,9 @@
     {{yield
       (hash
         label=(component control.label for=control.inputId srOnly=srOnly label=label)
-        input=(component 'bootstrap/-input'
+        input=(component 'bootstrap/-checkbox'
           checked=isChecked
-          onClick=(action "change")
+          onclick=(action "change")
```

Comment:
There is no `change` action in this component. This should be spitting an error into your console.